### DISCUSSION TITLE
when I used mixcr to analyze RNAseq data, there was an error "unknown options"

### DISCUSSION THREAD

**Discussion by Henry-zhang-For** [milaboratory/mixcr#1725] [Q&A] [answered]
I analyze RNAseq for TCR CDR3, use the code:
mixcr analyze shotgun --species hs --starting-material rna --receptor-type TCR 'HD1_1.fq.gz'
there is an error:
IMPORTANT: MiXCR will use at most 12000MB of RAM,
           use -Xmx to change automatically set heap size (i.e. -Xmx50g to set heap size to 50 Gb)
App version: 4.6.0; built=Sun Dec 10 03:48:42 CST 2023; rev=c9fafa41fe; lib=repseqio.v4.0
Unknown options: '--starting-material', '--receptor-type'
Possible solutions: --strict-sample-sheet-matching

**Comment by mizraelson** [comment, marked as answer]
Hi, please use the following command:

```
mixcr analyze rna-seq \
    --species hsa \
      input_R1.fastq.gz \
      input_R2.fastq.gz \
      result
```
